cmk/gui/plugins/wato/check_parameters/job.py

```diff
@@ -50,6 +50,17 @@ def _parameter_valuespec_job():
                  Age(title=_("Critical at"), default_value=0)
              ],
          )),
+        ("exit_codes",
+         ListOf(
```

Comment:
Add the argument allow_empty=False to the ListOf valuespec to prevent empty lists